dev/com.ibm.ws.kernel.boot.ws-server/publish/bin/server

```diff
@@ -1072,7 +1072,7 @@ javaCmd()
     # We direct STDERR to /dev/null to avoid that noise.
     {
       #  Launch with stdin set to /dev/null to avoid file-handle complications on criu restore
-      "${JAVA_CMD}" "$@" >> "${CHECKPOINT_CONSOLE_LOG}" 2>&1 < /dev/null
+      ld.so --glibc-hwcaps-mask "" "${JAVA_CMD}" "$@" >> "${CHECKPOINT_CONSOLE_LOG}" 2>&1 < /dev/null
```

Comment:
Won't this negate error handling on platforms that can get to this code but where  ld.so does not exist (like macOS)? Today they would get a checkpoint not supported message.

Reply:
Yes, this is a problem.  We decided to instead add a prefix variable that can be set externally to prefix the JAVA_CMD